http/limit/limit.go

```diff
@@ -31,18 +37,23 @@ func PerDays(decayDays, maxAttempts int) http.Limit {
 type Limit struct {
 	// The rate limit signature key.
 	Key string
-	// The maximum number of attempts allowed within the given number of minutes.
-	MaxAttempts int
-	// The number of minutes until the rate limit is reset.
-	DecayMinutes int
+	// The store instance.
+	Store limiter.Store
 	// The response generator callback.
 	ResponseCallback func(ctx http.Context)
 }
 
 func NewLimit(maxAttempts, decayMinutes int) *Limit {
+	store, err := memorystore.New(&memorystore.Config{
```

Comment:
Using memory? We can use redis before. Does `sethvargo/go-limiter` support Redis?

Reply:
> Using memory? We can use redis before. Does `sethvargo/go-limiter` support Redis?

Yes, If need, we can implement `Store` interface for redis.

Reply:
> > Using memory? We can use redis before. Does `sethvargo/go-limiter` support Redis?
> 
> Yes, If need, we can implement `Store` interface for redis.

Implemented